go.mod

```diff
@@ -26,7 +26,7 @@ require (
 	github.com/mitchellh/osext v0.0.0-20151018003038-5e2d6d41470f // indirect
 	github.com/ncw/swift v1.0.47
 	github.com/opencontainers/go-digest v1.0.0
-	github.com/opencontainers/image-spec v1.0.2
+	github.com/opencontainers/image-spec v1.1.0-rc2.0.20221013174636-8159c8264e2e
```

Comment:
Can we pin to `v1.1.0-rc2` instead of a commit?

Reply:
Done.